package.json

```diff
@@ -87,8 +88,10 @@
     "**/*.{js,ts}": "eslint --fix"
   },
   "dependencies": {
+    "@ungap/structured-clone": "1.2.0",
```

Comment:
The bundlephobia size of this polyfill looks reasonably small, but have you ran `npm run size` after making your changes using picocolors and the `structuredClone` polyfill? I'm curious if adze will become larger or smaller.

Reply:
I's not matching with the [bundlephobia](https://bundlephobia.com/package/adze@2.0.12) metrics. I guess size-limit calculates things differently. So I ran it against `master` to compare:

master = Size: 13.29 kB
picocolors = Size: 12.62 kB